packages/react/src/SteppedUpContent.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactNode, useEffect, useRef } from 'react';
+
+import { useStepUp, useIsSteppedUp, useIsAuthenticated } from '@frontegg/react-hooks';
+
+export interface SteppedUpProps {
+  maxAge?: number;
+  render?: (isSteppedUp: boolean) => React.ReactNode | null;
```

Comment:
Why not using a fallback instead? I think this is also good, might even give more flexibility for the clients
But fallback will be more clear 
@doregg @yuvalotem1

Reply:
what do u mean by fallback? where? @rotemzif1